logstash-output-pagerduty.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-output-pagerduty'
-  s.version         = '1.0.0'
+  s.version         = '1.1.0'
```

Comment:
Adding tests where there we no one is not a patch, I'm sorry! A patch is a fix for an already implemented feature.